src/thunks/index.js

```diff
@@ -0,0 +1,39 @@
+import { addNotes } from "../actions";
+import { fetchCall } from "../utils/api";
+
+const baseUrl = "http://localhost:3001/notes";
+
+export const getOptions = (method, data) => ({
```

Comment:
As we discussed, I really like this getOptions method.  I think that it works better here than in the api file. Nice work!